src/dokan/ceph_dokan.cc

```diff
@@ -784,10 +784,10 @@ static NTSTATUS WinCephGetVolumeInformation(
   DWORD FileSystemNameSize,
   PDOKAN_FILE_INFO DokanFileInfo)
 {
-  // TODO: configurable volume name and serial number.
-  // We should also support having multiple mounts.
-  wcscpy(VolumeNameBuffer, L"Ceph");
-  *VolumeSerialNumber = 0x19831116;
+  g_cfg->win_vol_name.copy(VolumeNameBuffer, VolumeNameSize);
+
+  // TODO: consider prepending this value with the cluster fsid.
+  *VolumeSerialNumber = ceph_get_fs_cid(cmount);
```

Comment:
@batrick I'd like your opinion on this one. Do you think we should fetch the cluster fsid and use that together with the ``ceph_get_fs_cid`` return value to construct a unique id? The ``fs_cluster_id_t`` value seems to be an index that starts from 0.

Reply:
You mean mounting different filesystems at the same time, right? The ``--client_fs`` option works, I've mentioned it here:  https://github.com/ceph/ceph/pull/39939/commits/033b936755ebb96105bdad8aaa55f5fb7b22e7cd

Worth mentioning that the ceph-dokan daemon doesn't currently support running in the background. One idea is to use NSSM to run one or more ceph-dokan daemons as Windows services. We might eventually use a centralized service like we do for RBD: https://github.com/ceph/ceph/blob/master/README.windows.rst#windows-service.

Reply:
> You mean mounting different filesystems at the same time, right? The `--client_fs` option works, I've documented it here: [033b936](https://github.com/ceph/ceph/commit/033b936755ebb96105bdad8aaa55f5fb7b22e7cd)

I mean mounting different paths in CephFS.

Reply:
Ohh, I see. Multiple subdirectories can be mounted at the same time using the "--root-path" parameter. Here's an example: http://paste.openstack.org/raw/803417/

In that case, we'd have multiple mounts with the same volume name and serial number. Windows and Dokan allow it but it might confuse a few applications, depending on how they use the fs id.

On the other hand, just like the Windows docs state [1], the file id and volume id could then be used to check if two handles point to the same file. 

[1] https://docs.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-file_id_info